src/components/forms/FormFactory.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormGroup, Col, ControlLabel, HelpBlock, Form, FieldLevelHelp } from 'patternfly-react';
+import { get, has } from 'lodash';
+import { TextArea, Dropdown, Checkbox, Text } from '.';
+
+export const FormFactory = ({ fields, fieldsValues, onFormChange }) => {
+  const formGroups = Object.keys(fields)
+    .filter(key => !(fields[key].isVisible && fields[key].isVisible(fieldsValues) === false))
```

Comment:
nitpick to be generic:
change `fields[key].isVisible(fieldsValues)` to `!!fields[key].isVisible(fieldsValues)`

Reply:
fixed